flexy-osisoft-connector/src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -189,6 +189,40 @@ public int initTags(ArrayList tagList) throws JSONException{
       return retval;
    }
    
+   public void postTagsLive(ArrayList tags)
+   {
+      //HTTPS responses are stored in this file
+      String filename = "/usr/response.txt";

```

Comment:
concern that we might add a lot of wear on the flash? do we have to write the response back to a file each time or can we get it back in memory? is there a filesystem location that is mounted in ram like /tmp that we could use instead if we have to write it back to a file upon receiving it?

Reply:
The RequestHttpX method takes a filename only to store the response to.  I am not aware of anywhere other than /usr/ that we have rights to on the Flexy.  Ill ask Deryck if he is available if there is a better option.  

We could also omit the filename now but that is where OSIsoft will tell us about issues if a tag value was not posted but the connection was successful.

Reply:
@cmo-hms removed writing to the file for now since its currently unused (it was used for debugging and left in because it will eventually get used (maybe) for error logging and retries)